libadikted/Makefile

```diff
@@ -1,14 +1,50 @@
 # Project: libadikted
+# Makefile created by KeeperFX/dkfans team
 
-#To create a debug executable, run with the command:
-# make DEBUG
+# Instructions:
+# To create the Windows library run:
+# make all
+# To create the linux library run:
+# make all BUILD_FOR_LINUX=1 (not fully implemented)
+# To create a debug library, run with the command:
+# make DEBUG (THIS IS NOT IMPLEMENTED YET)
+# To clean up the built files (objects, library, and resource file) run:
+# make clean
 
-#CROSS_COMPILE = i686-w64-mingw32-
+# Tested and working with:
+# - Windows : MSYS2 (MinGW 32-bit) - make
+# - Windows : MSYS2 (MinGW 32-bit) - mingw32-make
+# - Windows : MinGW (MSYS) - make
 
-
-CC           = $(CROSS_COMPILE)gcc
-LINK         = $(CROSS_COMPILE)gcc
-SRC          = \
+# Executable files extension on host environment
+ifneq (,$(findstring Windows,$(OS)))
+  CROSS_EXEEXT = .exe
+else
+  CROSS_EXEEXT =
+  CROSS_COMPILE = i686-w64-mingw32-
+endif
+# Executable files extension on target environment (
+#BUILD_FOR_LINUX = 1
+ifneq (,$(BUILD_FOR_LINUX))
+  LIBEXT = .so
+  OTHERLIBEXT = .dll
+else
+  LIBEXT = .dll
+  OTHERLIBEXT = .so
+endif
+# Names of utility commands
+CPP                = $(CROSS_COMPILE)g++$(CROSS_EXEEXT) -std=c89
```

Comment:
Can we remove this and the similar line in `mapslang/Makefile`?
g++ is not needed/used.
Or is it a good idea to keep them in case they are needed in the future?

Reply:
Remove it